BSP/Simulator/Hardware/Timer.py

```diff
@@ -0,0 +1,37 @@
+import csv
+import os
+
+# Path of file
+file = "Data/Timer.csv"

```

Comment:
Make this the full file path like in Switches.py since the script will be executed from the root directory using a symbolic link.